### ISSUE TITLE
WCAG 1.1.1: Ensures <img> elements have alternate text or a role of none or presentation (img[height="\35 0"])

### ISSUE THREAD

**Issue by VitthalBobade** [VitthalBobade/accessibilityinsightsreports#3]
#### Issue
`Ensures <img> elements have alternate text or a role of none or presentation` ([image-alt](https://dequeuniversity.com/rules/axe/3.5/image-alt?application=msftAI))
#### Target application
[BuddyConnect Admin Console | Login](https://buddyconnectwebdev.azurewebsites.net/)
#### Element path
img[height="\35 0"]
#### Snippet
`<img src="/Content/img/Login_BuddyConnectlogo.svg" height="50">`
#### How to fix
    Fix any of the following:
      Element does not have an alt attribute
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute or the title attribute is empty
      Element's default semantics were not overridden with role="presentation"
      Element's default semantics were not overridden with role="none"
#### Environment
Chrome version 84.0.4147.105
This accessibility issue was found using Accessibility Insights for Web 2.20.0 (axe-core 3.5.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
